tests/slack-hook-handler.spec.js

```diff
@@ -55,6 +55,18 @@ describe("SlackHookHandler.handle", function() {
             expect(intent.sendText).toHaveBeenCalledWith(
                 "!room:host", "<special & characters>");
         });
+        it("unicode-ifies emoji", function() {
+            intent = createSpyObj("intent", ["sendText"]);
+            handler.handle(makeRequest(
+                    "Lucille",
+                    "nudge nudge:wink: :wink::rugby_football:wink :wink wink:"
+                )
+            );
+            expect(intent.sendText).toHaveBeenCalledWith(
+                "!room:host",
+                "nudge nudge😉 😉🏉wink :wink wink:"
```

Comment:
It is *extremely* difficult to tell that this should be the expected output. It would be nicer to have multiple tests to assert that it:
 - Can do multiple `:foo:` on a single line
 - Doesn't do `:fo o:` (whitespace). Doesn't do ones with additional `:`s e.g. `:fo:o:`.
 - Gracefully handles unknown strings like `:nomnomnom:` by leaving them alone.